src/dashboard/src/components/administration/forms.py

```diff
@@ -432,49 +243,81 @@ class ProcessingConfigurationForm(forms.Form):
         'initial': None,
     }
 
+    # This extends the fields defind in the processing_config module.
+    # Temporary solution until workflow data can be translated.
+    LABELS = {
+        'bd899573-694e-4d33-8c9b-df0af802437d': _('Assign UUIDs to directories'),
+        '755b4177-c587-41a7-8c52-015277568302': _('Send transfer to quarantine'),
+        '19adb668-b19a-4fcb-8938-f49d7485eaf3': _('Remove from quarantine after (days)'),
+        '56eebd45-5600-4768-a8c2-ec0114555a3d': _('Generate transfer structure report'),
+        'f09847c2-ee51-429a-9478-a860477f6b8d': _('Select file format identification command (Transfer)'),
+        'dec97e3c-5598-4b99-b26e-f87a435a6b7f': _('Extract packages'),
+        'f19926dd-8fb5-4c79-8ade-c83f61f55b40': _('Delete packages after extraction'),
+        '70fc7040-d4fb-4d19-a0e6-792387ca1006': _('Perform policy checks on originals'),
+        'accea2bf-ba74-4a3a-bb97-614775c74459': _('Examine contents'),
+        'bb194013-597c-4e4a-8493-b36d190f8717': _('Create SIP(s)'),
+        '7a024896-c4f7-4808-a240-44c87c762bc5': _('Select file format identification command (Ingest)'),
+        'cb8e5706-e73f-472f-ad9b-d1236af8095f': _('Normalize'),
+        'de909a42-c5b5-46e1-9985-c031b50e9d30': _('Approve normalization'),
+        '498f7a6d-1b8c-431a-aa5d-83f14f3c5e65': _('Generate thumbnails'),
+        '153c5f41-3cfb-47ba-9150-2dd44ebc27df': _('Perform policy checks on preservation derivatives'),
+        '8ce07e94-6130-4987-96f0-2399ad45c5c2': _('Perform policy checks on access derivatives'),
+        '05357876-a095-4c11-86b5-a7fff01af668': _('Bind PIDs'),
+        'd0dfa5fc-e3c2-4638-9eda-f96eea1070e0': _('Document empty directories'),
+        'eeb23509-57e2-4529-8857-9d62525db048': _('Reminder: add metadata if desired'),
+        '7079be6d-3a25-41e6-a481-cee5f352fe6e': _('Transcribe files (OCR)'),
+        '087d27be-c719-47d8-9bbb-9a7d8b609c44': _('Select file format identification command (Submission documentation & metadata)'),
+        '01d64f58-8295-4b7b-9cab-8f1b153a504f': _('Select compression algorithm'),
+        '01c651cb-c174-4ba4-b985-1d87a44d6754': _('Select compression level'),
+        '2d32235c-02d4-4686-88a6-96f4d6c7b1c3': _('Store AIP'),
+        'b320ce81-9982-408a-9502-097d0daa48fa': _('Store AIP location'),
+        '92879a29-45bf-4f0b-ac43-e64474f0f2f9': _('Upload DIP'),
+        '5e58066d-e113-4383-b20b-f301ed4d751c': _('Store DIP'),
+        'cd844b6e-ab3c-4bc6-b34f-7103f88715de': _('Store DIP location'),
+    }
```

Comment:
We have to do this until i18n makes it into workflow data. Ugly but just a temporary hack, soon to be fixed as part of #1101.